alot/account.py

```diff
@@ -214,6 +214,9 @@ def errb(failure):
             logging.error(failure.value.stderr)
             raise SendingMailFailed(errmsg)
 
+        # make sure self.mail is a string
+        mail = str(mail)
```

Comment:
I'm doubtful that the email lib in python2 does the right thing. However, you are correct that this is equivalent to the code it's replacing, so it's not a regression. Handling unicode correctly in python2 is hard, so it's probably fine to leave it until python3 arrives.